cmd/podman/commit.go

```diff
@@ -83,7 +84,7 @@ func commitCmd(c *cli.Context) error {
 			return errors.Errorf("messages are only compatible with the docker image format (-f docker)")
 		}
 	case "docker":
-		mimeType = buildah.Dockerv2ImageManifest
+		mimeType = manifest.DockerV2Schema1MediaType
```

Comment:
I think you want manifest.DockerV2Schema2MediaType here.....

'''
image/manifest/manifest.go
        // DockerV2Schema1MediaType MIME type represents Docker manifest schema 1
        DockerV2Schema1MediaType = "application/vnd.docker.distribution.manifest.v1+json"
        // DockerV2Schema2MediaType MIME type represents Docker manifest schema 2
        DockerV2Schema2MediaType = "application/vnd.docker.distribution.manifest.v2+json"

Buildah side
image.go:	Dockerv2ImageManifest = docker.V2S2MediaTypeManifest
./docker/types.go:const V2S2MediaTypeManifest = "application/vnd.docker.distribution.manifest.v2+json"

Reply:
yup, @wking pointed this out